packages/esm-form-engine-app/src/form-renderer/form-renderer.scss

```diff
@@ -16,3 +15,20 @@
 .errorContainer {
   height: 100vh;
 }
+
+.printButton {
+  display: flex;
+  margin-left: auto;
+}
+
+.sidebar > {
+  .error {
+    color: red;
+  }
+}
+
+@media print {
+  :global(.cds--btn-set) {
+    display: none;
+  }
```

Comment:
Makes it so that we're not including the on-screen action buttons in the printout.